configurator/src/main/java/org/picmg/configurator/StateSetTabController.java

```diff
@@ -84,30 +61,97 @@ public class StateSetTabController implements Initializable {
 	@FXML private ImageView outputUnitsImage;
 	@FXML private ImageView plusAccuracyImage;
 
+	StateSetTabController.StateSetTableData workingData = new StateSetTableData();
+	boolean modified;
```

Comment:
What is the aim of this variable? I only see it set to

Reply:
Removed modified as  we are not using this currently